frontend/src/components/Cards/CardStore.js

```diff
@@ -5,27 +5,26 @@ import Button from 'react-bootstrap/Button';
 import GameInstance from "../../services/gameService";
 
 class CardStore extends React.Component {
-    constructor(props, context) {
-        super(props, context);
-
-        this.handleChange = this.handleChange.bind(this);
-        this.clearSelected = this.clearSelected.bind(this);
-
-        this.state = {
-            username: props.currentUser,
-            gameRoom: props.currentRoom,
-            value: [],
-            selectedCard: [],
-            cardName: 'Card Name',
-            cardCost: 3,
-            cardType: 'keep',
-            cardEffect: 'lose two hearts, gain 4 energy',
-            cardFootnote: 'footnote description',
-
-        };
-    }
+  constructor(props, context) {
+    super(props, context);
+
+    this.handleChange = this.handleChange.bind(this);
+    this.clearSelected = this.clearSelected.bind(this);
+
+    this.state = {
+      username: props.currentUser,
+      gameRoom: props.currentRoom,
+      value: [],
+      selectedCard: [],
+      card: [
+        { name: 'Armor Plating', cost: 4, type: 'Keep', effect: 'Ignore damage of 1', footnote: ' ' },
```

Comment:
I don't understand why these are here. Don't the cards get generated now? Wouldn't that mean we don't need these hard coded values?

Reply:
I added those hard coded values before changes were made to engine/board.py. Removed those values